project/project.json5

```diff
@@ -173,16 +161,100 @@
                         base: ['res', 'com'],
                     },
                     subsector: {
-                        base: ['unspecified'],
```

Comment:
Per the above, I think this change should be reverted, which will require adding a mapping file to map all the tilt-azimuth combinations (at the dataset level) to "unspecified" (at the project level).